examples/llm/mcloud/mcli-1b-eval.yaml

```diff
@@ -0,0 +1,69 @@
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  git_branch: feature/composer_icl_eval_ready_to_merge
+  # git_commit: # OR use your commit hash
+  pip_install: -e .[llm]
+  ssh_clone: false
+
+
+# We are fetching, converting, and training on the 'val' split
+# as it is small and quick to get going for this demo.
+# For real training runs, follow the instructions in `examples/llm/README.md`
+# to convert and host the full 'train' dataset.
+command: |
+  cd examples/examples/llm
+  python -m icl_eval.evaluate_model /mnt/config/parameters.yaml
```

Comment:
is it intentional for this to be `python` here and `composer` in the other MCLOUD file?

Reply:
Just lmk and I can clean it up!

Reply:
good catch!